prediction_market_agent/agents/microchain_agent/app.py

```diff
@@ -204,12 +235,25 @@ def get_function_bullet_point_list(model: str) -> str:
         "To start, click 'Run' to see the agent in action, or bootstrap the "
         "agent with your own reasoning."
     )
-    st.markdown(
-        f"It is equipped with the following tools to access the "
-        f"[{MARKET_TYPE}]({MARKET_TYPE.market_class.base_url}) prediction "
-        f"market APIs:"
-    )
-    st.markdown(get_function_bullet_point_list(model=model))
+    st.markdown("It is equipped with the following tools:")
+    if agent_is_initialized():
+        st.markdown(
+            get_function_bullet_point_list(agent=st.session_state.agent, model=model)
+        )
+    else:
+        st.markdown("The agent is not initialized yet.")
+
+with st.expander("Agent's current system prompt"):
+    if agent_is_initialized():
+        st.markdown(st.session_state.agent.system_prompt)
```

Comment:
If we've already got the 'info' dropdown box that states the tools the agent has, then I think it would look nicer if this only contained the system prompt above `NON_UPDATABLE_DIVIDOR`, so we don't have duplicate-ish info

Reply:
Sounds good, @coderabbitai please create an issue for this